pypeclub/OpenPype
# Idle manager prevent to stop tray

## Issue
Idle manager have running threads on tray exit which cause not ending process of tray. Worst is that it's visible only when using OpenPype from code or with `openpype_console` executable.

## Changes
- stop all running threads on idle manager stop so the process should end properly

This happens on Linux.

### How to test (maybe?)
- start `openpype_console` executable (so you can see if it's closed)
- make sure idle manager has registered callbacks (Ftrack enabled and logged, Timers Manager enabled)
    - probably start timer on Ftrack (?)
- run `Exit` in tray
- whole Pype process should shutdown (console dissappear)

[cuID:yjx9d4]
